panels/_locators.py

```diff
@@ -109,10 +109,25 @@ def figsize_in(self, units):
         return (convert_units(self.figwidth, self.units, units),
                 convert_units(self.figheight, self.units, units))
 
-    def panel_position_iterator(self):
-        """Returns a generator of panel positions."""
-        return (self.panel_position(r, c)
-                for r in range(self.rows) for c in range(self.columns))
+    def panel_position_iterator(self, major='row'):
+        """Returns a generator of panel positions.
+
+        Keyword argument:
+
+        major (default='row'): str
```

Comment:
Put a `*` in-front of this so it renders as a bullet list in the documentation (see other docstrings).